docs/modules/azure.md

```diff
@@ -5,14 +5,95 @@ This module is INCUBATING. While it is ready for use and operational in the curr
 
 Testcontainers module for the Microsoft Azure's [SDK](https://github.com/Azure/azure-sdk-for-java).
 
-Currently, the module supports `CosmosDB` emulator. In order to use it, you should use the following class:
+Currently, the module supports `Azurite` and `CosmosDB` emulators. In order to use them, you should use the following classes:
 
 Class | Container Image
 -|-
+AzuriteContainer | [mcr.microsoft.com/azure-storage/azurite](https://github.com/microsoft/containerregistry)
 CosmosDBEmulatorContainer | [mcr.microsoft.com/cosmosdb/linux/azure-cosmos-emulator](https://github.com/microsoft/containerregistry)
 
 ## Usage example
 
+### Azurite Storage Emulator
+
+#### Using Blobs
+
+Start Azurite Emulator during a test using Blob functionality:
+
+<!--codeinclude-->
+[Starting a Azurite Blob container](../../modules/azure/src/test/java/org/testcontainers/containers/AzuriteContainerTest.java) inside_block:blobEmulatorContainer
+<!--/codeinclude-->
+
+Get the connection string from the container:
+
+<!--codeinclude-->
+[Get connection string](../../modules/azure/src/test/java/org/testcontainers/containers/AzuriteContainerTest.java) inside_block:getBlobConnectionString
+<!--/codeinclude-->
+
+Build Azure Blob client:
+
+<!--codeinclude-->
+[Build Azure Blob Service client](../../modules/azure/src/test/java/org/testcontainers/containers/AzuriteContainerTest.java) inside_block:createBlobClient
+<!--/codeinclude-->
+
+Test against the Emulator:
+
+<!--codeinclude-->
+[Testing against Azurite container](../../modules/azure/src/test/java/org/testcontainers/containers/AzuriteContainerTest.java) inside_block:testWithBlobClient
+<!--/codeinclude-->
```

Comment:
I don't think we need to document this.

Reply:
Thank you! I have followed the CosmosDB example: https://github.com/testcontainers/testcontainers-java/blob/9cca5368343342e149bc452c473c52445637466c/docs/modules/azure.md?plain=1#L123-L127
Should I remove it from that as well for the sake of consistency or is it fine if I am just not adding the new ones?

Reply:
Let's focus on the changes introduced in this PR

Reply:
Done, thank you!